charts/pathfinder-rpc-server/templates/ingress.yaml

```
@@ -0,0 +1,27 @@
+---    
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: pathfinder-ingress
+  namespace: pathfinder
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/router.middlewares: pathfinder-corsheaders@kubernetescrd 
```

Above the middleware doesn't hold the prefix `pathfinder-`; is that one assumed from the Helm installation? Then this could be documented within a README.

If the prior holds true, and the previous suggestion is accepted, then we could possibly write:

```suggestion
    traefik.ingress.kubernetes.io/router.middlewares: pathfinder-cors-headers@kubernetescrd 
```

---

this is more from traefik side for ingress, it recognises de middleware by `namespace-middlerwareName`

---

My eye is really used to seeing the letters C O R S touching each and no other, clearly delimited from other words.

Did it work with hyphen?